pkgs/ipxe/default.nix

```diff
@@ -1,4 +1,15 @@
-{ stdenv, writeText, perl, xz, openssl, src, lib }:
+{
+  efi ? false,
```

Comment:
it feels like nothing speaks against enabling both by default as it doesn't add up to more deps in either way, right?